### DISCUSSION TITLE
ineligible transactions for github copilot

### DISCUSSION THREAD

**Discussion by benyamin-nayudi** [community/community#73019] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

i've got an issue with buying a copilot license, I am living in Iran and under the U.S law any transactions from iran has been baned! and we do not have any access to anything in the world due to this restrictions...

i have entered my bank account wrongly when I was In iran and github blocked my account so I can't buy copilot and this is the message that is showing for me:
"""
Cannot proceed with payment
We are sorry, but after reviewing we have determined that the billing information supplied is ineligible for transactions with GitHub due to government restrictions.
[Learn more](https://docs.github.com/articles/github-and-trade-controls) about GitHub and trade control regulations.
"""
now I have migrated to another country but this blocking is still there, how can I reset so I can but copilot?

**Comment by davecheney** [comment, marked as answer]
@benyamin-nayudi  please contact support who will be able to assist you.

**Reply by benyamin-nayudi** [reply]
i am exaclly trying to find someone to assist me here, because there wasn't any link to any support, just the community, no emails no phone number nothing.

**Reply by davecheney** [reply]
https://support.github.com/contact

**Reply by salazar94** [reply]
> i am exaclly trying to find someone to assist me here, because there wasn't any link to any support, just the community, no emails no phone number nothing.

Sorry for this question, but did support help you solve the problem?

**Reply by movalex** [reply]
Support don't answer change billing address requests. I have my submitted over a month ago, no trace that they even care. Had to create a new account to enter my new billing credentials.

**Reply by 713b76b7-2247-4bd5-98a7-014226363f9b** [reply]
> Support don't answer change billing address requests. I have my submitted over a month ago, no trace that they even care. Had to create a new account to enter my new billing credentials.

same here🥲, i even attach my Gov ID to prove that my country of residence has changed

**Comment by DosX-dev** [comment]
I have the same problem. I can't install extensions because I have incorrect payment information. GitHub doesn't have official support for users, so I still can't solve this problem
